pkg/qemu/qemu.go

```diff
@@ -425,6 +427,7 @@ func Cmdline(cfg Config) (string, []string, error) {
 
 	// QEMU process
 	args = append(args, "-name", "lima-"+cfg.Name)
+	args = append(args, "-uuid", uuid.NewMD5(uuid.NameSpaceOID, []byte(cfg.Name)).String() )
```

Comment:
For universal uniqueness, can we generate a random UUID and save it as “uuid” file in the instance dir?
https://github.com/lima-vm/lima/blob/3aca16d2a762fdb1ba9f50d6749cb03240d0a963/pkg/store/filenames/filenames.go#L25-L45

Reply:
( no need to save the uuid as a file if you adopt Jan’s approach https://github.com/lima-vm/lima/pull/754#issuecomment-1077774078 )